gbpservice/neutron/nsf/db/nsf_db_model.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2016 OpenStack Foundation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oslo_db.sqlalchemy import models
+from oslo_utils import uuidutils
+import sqlalchemy as sa
+from sqlalchemy.ext import declarative
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import orm
+
+
```

Comment:
If the DB migration has to be part of GBP then they have to be part of Neutron DB